tide_site.module

```diff
@@ -588,6 +588,40 @@ function tide_site_form_alter(array &$form, FormStateInterface $form_state, $for
       }
     }
   }
+  // Add site prefix to redirect URL.
+  if ('redirect_redirect_form' === $form_id) {
+    $sites_term = 'sites';
+    // Get all primary sites ony, do not load sub-sites.
+    $term_tree = \Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadTree(
+      $sites_term,
+      0,
+      1,
+      TRUE
+    );
+    // Build options list.
+    $options = [];
+
+    foreach ($term_tree as $term) {
+      if ($term instanceof Term) {
+        $options['site-' . $term->id()] = $term->getName();
+      }
+    }
+    asort($options, SORT_STRING);
+    $options = [...['none' => 'Select site'], ...$options];
```

Comment:
Change `none` to `0` for the first Site in the drop-down. Use an integer.

Change the `Select site` to `None` or `-` (dash). If a user chooses to not select a site, they're wanting to create a redirect for the CMS. This is allowed. You might want to redirect from a Drupal admin page to another Drupal admin page, so it is okay to not "Select site" but just use the default of no site.